<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="loginPage">
      <widget class="QLabel" name="userIdLabel">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>260</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>User id:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="createAccountPushButton">
       <property name="geometry">
        <rect>
         <x>259</x>
         <y>320</y>
         <width>171</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>create new account</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>290</y>
         <width>261</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="userIdLineEdit">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>260</y>
         <width>261</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="passwordLabel">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>290</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loginPushButton">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>320</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="createUser">
      <widget class="QLabel" name="setUserNameLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>170</y>
         <width>71</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>User Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="setUserNameLineEdit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>170</y>
         <width>221</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="setUserIdLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>210</y>
         <width>71</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>User Id:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="setUserIdLineEdit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>210</y>
         <width>221</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="setPasswordLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>250</y>
         <width>71</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="setPasswordLineEdit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>250</y>
         <width>221</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="confirmPasswordLabel">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>290</y>
         <width>111</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Confirm Password:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="confirmPasswordLineEdit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>290</y>
         <width>221</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="savePushButton">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>370</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>save</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="setRoleLineEdit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>330</y>
         <width>221</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="setRoleLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>330</y>
         <width>71</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Role:</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
